entries/s/surveymonkey.com.json

```
@@ -0,0 +1,13 @@
+{
+  "SurveyMonkey": {
+    "domain": "surveymonkey.com",
+    "contact": {
+      "twitter": "SurveyMonkey",
+      "form": "https://help.surveymonkey.com/en/contact/"
+    },
+    "categories": [
+      "marketing",
+      "other"
```

As the `other` category is only for entries that don't fit in any other category, it's probably best to just use the `marketing` category.
```suggestion
      "marketing"
```